translations/en-us.yml

```diff
@@ -1418,6 +1422,9 @@ registries:
         justificationForWithdrawal: 'Justification for Withdrawal'
         justificationForWithdrawalPlaceholder: 'Provide justification for withdrawal'
         noModeratorActions: 'No moderator decisions can be made at this time'
+    myRegistrationsList:
+        noRegistrations: "You don't have any registrations"
+        noDrafts: "You don't have any drafts"
```

Comment:
I think we avoid using contractions like "don't, can't, won't, etc" for user facing language, so `You do not have any registrations/drafts` here would be better